File: Source/Addins/Eto.Addin.VisualStudio/Eto.ProjectTemplates/VB/Library/MainForm.jxeto.vb
Imports Eto.Forms
Imports Eto.Drawing
Imports Eto.Serialization.$SerializationType$

Public Class MainForm
	Inherits Form

	Public Sub New()
		$SerializationType$Reader.Load(this)
	End Sub

	Sub HandleQuit(ByVal sender As Object, ByVal e As EventArgs)
		Application.Instance.Quit()
	End Sub

	Sub HandleClickMe(ByVal sender As Object, ByVal e As EventArgs)
		MessageBox.Show(Me, "I was clicked!")
	End Sub

End Class

File: src/Addins/Eto.Addin.VisualStudio/Eto.ProjectTemplates/CS/MainForm.jxeto.cs
using System;
using System.Collections.Generic;
using Eto.Forms;
using Eto.Drawing;
using Eto.Serialization.$SerializationType$;

namespace $safeprojectname$
{	
	public class MainForm : Form
	{	
		public MainForm()
		{
			$SerializationType$Reader.Load(this);
		}

		protected void HandleClickMe(object sender, EventArgs e)
		{
			MessageBox.Show("I was clicked!");
		}

		protected void HandleQuit(object sender, EventArgs e)
		{
			Application.Instance.Quit();
		}
	}
}
